src/HexagonOptimize.cpp

```diff
@@ -780,6 +788,8 @@ class OptimizePatterns : public IRMutator {
             { "halide.hexagon.pack_satb.vh", i8_sat(wild_i16x) },
             { "halide.hexagon.pack_sath.vw", i16_sat(wild_i32x) },
 
+            { "halide.hexagon.trunc_satuh.vuw", u16_sat(wild_u32x), Pattern::DeinterleaveOp0 | Pattern::v62 },
```

Comment:
Add comment here:

// We don't have a vpack equivalent to this one, so we match it directly.